packages/sources/uscpi-one/README.md

```diff
@@ -0,0 +1,49 @@
+# Chainlink External Adapter for US Consumer Price Index (USCPI)
+
+---
+
+### Input Parameters
+
+| Required? |   Name   |     Description     |         Options          | Defaults to |
+| :-------: | :------: | :-----------------: | :----------------------: | :---------: |
+|           | endpoint | The endpoint to use | [serie](#Serie-Endpoint) |    serie    |
+
+---
+
+## Serie Endpoint
```

Comment:
The README says the endpoint name is "serie" but the code uses endpoint name "series"